tests/test_formulagrader.py

```diff
@@ -492,57 +492,61 @@ def test_fg_debug_log():
             'z': ComplexRectangle()
         },
         blacklist=['sin', 'cos'],
-        user_functions={'f': RandomFunction()},
+        user_functions={
+            'f': RandomFunction(),
+            'square': lambda x: x**2
+            },
         samples=2,
         debug=True
     )
     result = grader(None, 'z + x*x + f(y)')
 
     message = (
-        "<pre>MITx Grading Library Version {version}<br/>\n"
+        "<pre>MITx Grading Library Version 1.1<br/>\n"
```

Comment:
Surely you want this to be read from the version file?

Reply:
Yes...I keep copying from terminal output and forgetting to change that back.

Do you want this as 1.1.1 in the changlong =/

Reply:
No worries - I don't think this broke anything. We'll bundle some more work together and put out 1.1.1.